example/demo_v5.rb

```diff
@@ -54,4 +54,9 @@
 puts value
 puts
 
+puts "Show audit entries for previous actions"
```

Comment:
@uCatu how does it know what is the "previous actions"? does it just dumps all the audit since the beginning?

Reply:
Yes, it will show audit trail for all demo_v5 audited actions.